classes/mugoesitemplateoperators.php

```diff
@@ -42,8 +48,21 @@ function modify( $tpl, $operatorName, $operatorParameters, $rootNamespace, $curr
                 {
                     // Advertise this back to the reverse proxy
                     header( 'Surrogate-Control: abc=ESI/1.0' );
+                    ezjscPackerTemplateFunctions::setPersistentArray( 'surrogate', true, $tpl, false, false, false, true );
                     $widgetURL = 'esiwidget/view/(widget)/' . $widgetName;
                     eZURI::transformURI( $widgetURL );
+                    if(isset($namedParameters['widget_params']))
+                    {
+                        $widgetURL .= '?';
+                        $last = array_pop(array_keys($namedParameters['widget_params']));
+                        foreach($namedParameters['widget_params'] as $index => $param)
+                        {
+                            $widgetURL .= urlencode($index) . '=' . urlencode($param);
+                            if($index != $last) {

```

Comment:
Needs spacing to follow coding style and the curly bracket should be on a new line